packages/admin-next/dashboard/src/routes/invite/invite.tsx

```diff
@@ -0,0 +1,341 @@
+      <div className="flex w-full max-w-[300px] flex-col items-center">
+        <LogoBox
+          className="mb-4"
+          checked={success}
+          containerTransition={{
+            duration: 1.2,
+            delay: 0.8,
+            ease: [0, 0.71, 0.2, 1.01],
+          }}
+          pathTransition={{
+            duration: 1.3,
+            delay: 1.1,
+            bounce: 0.6,
+            ease: [0.1, 0.8, 0.2, 1.01],
+          }}
+        />
+        <div className="max-h-[557px] w-full will-change-contents">
+          <AnimatePresence>
+            {!success ? (
+              <motion.div
+                key="create-account"
+                initial={false}
+                animate={{
+                  height: "557px",
+                  y: 0,
+                }}
+                exit={{
+                  height: 0,
+                  y: 40,
+                }}
+                transition={{
+                  duration: 0.8,
+                  delay: 0.6,
+                  ease: [0, 0.71, 0.2, 1.01],
+                }}
+                className="w-full will-change-transform"
+              >
+                <motion.div
+                  initial={false}
+                  animate={{
+                    opacity: 1,
+                    scale: 1,
+                  }}
+                  exit={{
+                    opacity: 0,
+                    scale: 0.7,
+                  }}
+                  transition={{
+                    duration: 0.6,
+                    delay: 0,
+                    ease: [0, 0.71, 0.2, 1.01],
+                  }}
+                  key="inner-create-account"
+                >
+                  <CreateView onSuccess={() => setSuccess(true)} />
+                </motion.div>
+              </motion.div>
+            ) : (
+              <motion.div
+                key="success-view"
+                initial={{
+                  opacity: 0,
+                  scale: 0.4,
+                }}
+                animate={{
+                  opacity: 1,
+                  scale: 1,
+                }}
+                transition={{
+                  duration: 1,
+                  delay: 0.6,
+                  ease: [0, 0.71, 0.2, 1.01],
+                }}
+                className="w-full"
+              >
+                <SuccessView />
+              </motion.div>
+            )}
+          </AnimatePresence>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const CreateView = ({ onSuccess }: { onSuccess: () => void }) => {
+  const { t } = useTranslation()
+  const [searchParams] = useSearchParams()
+  const token = searchParams.get("token")
+
+  let tokenStatus: "valid" | "invalid" | "expired" = "valid"
+  const decodedToken: {
+    invite_id: string
+    role: UserRoles
+    user_email: string
+    iat: number
+  } | null = token ? decodeToken(token) : null
+
+  if (!decodedToken) {
+    tokenStatus = "invalid"
+  }
```

Comment:
Have you tested this with a token that is not a JWT token - e.g. some random characters. Doesn't seem to result in an invalid token status

Reply:
I did test it, and on my end it did fail when the token was not a valid JWT token, but I guess it was possible to have false positives, as long as decodeToken didn't return `null`. 

I have added a parse with zod, that validates that the decoded token has all the expected fields, e.g. a valid email, invite_id, etc, and if any one is missing then we show a invalid state screen instead 👍

Reply:
Nice! LGTM 💪